src/bin/qxmoji/main.c

```diff
@@ -1,3 +1,8 @@
+/* Needed to build on Linux */
+#ifdef __linux__
```

Comment:
Please also remove these (ifdefs and comments) ;-)

`_POSIX_C_SOURCE` is defined in the POSIX standard and should always be set to the version of the standard required. Just because FreeBSD's headers don't enforce it doesn't mean it's a "Linux thing" ;) I just goofed forgetting them...

Reply:
Fixed, the #ifdef magic was not needed at all, you are totally right about that.
After all, the POSIX macros are not Linux exclusive.